website/docs/advanced-concepts/granular-access-control/reference/default-roles.md

```diff
@@ -0,0 +1,663 @@
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                </tr>
+                <tr>
+                    <td>&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;Default Roles</td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                </tr>
+                <tr>
+                    <td>&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;Custom Roles</td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                </tr>
+                <tr>
+                    <td>**Workspace**</td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                </tr>
+                <tr>
+                    <td>**Audit Logs**</td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+
+For more information about each permission, see [Permissions](/advanced-concepts/granular-access-control/reference/permissions).
+
+### Limitations
+
+Despite having extensive permissions, the Instance Administrator role in Appsmith cannot perform the following tasks:
+
+- Inability to access or update individual workspaces or app content. This means that Instance Administrators cannot view or edit the specific content within each workspace or app, such as queries, datasources, or their configurations. 
```

Comment:
The instance administrator though has permission to assign any role to itself or to any user.  This means that if the instance admin does need to take over an individual workspace, they can assign the default workspace role to themselves or to any other user to get access.

Reply:
Leaving it here in case we want to add this as a caveat or a way to unblock the user if thats what they are looking for.

Reply:
We are just showing the configuration of the default role as is and what type of permissions are available. Do you think, we should add this information here that Instance administrators can assign any role to themselves?

Reply:
As discussed, we will not add the roles assignment information. Adding this info will confuse the users.